snuba/subscriptions/scheduler_processing_strategy.py

```diff
@@ -212,12 +213,15 @@ def __init__(
         self.__partitions = partitions
         self.__max_ticks_buffered_per_partition = max_ticks_buffered_per_partition
         self.__next_step = next_step
+        self.__metrics = metrics
 
         self.__buffers: Mapping[int, Deque[Message[Tick]]] = {
             index: deque() for index in range(self.__partitions)
         }
 
         self.__closed = False
+        self.__record_frequency_seconds = 180  # 3 minutes
```

Comment:
Why not record every minute? In Datadog, you would see the metric value change every 3 minutes with the current implementation. So you would delay the time it takes to gauge if there's a problem

Reply:
Updated to 60 seconds